emu/src/memory/interrupts.rs

```diff
@@ -49,7 +55,7 @@ impl Interrupts {
 
         Self {
             interrupt_enable: IORegister::with_access_control(ReadWrite),
-            interrupt_request: IORegister::with_access_control(ReadWrite),
+            interrupt_request: VecFixed::initialize(0),
```

Comment:
Why don't keep `new` for this?

Reply:
I wanted to initialize every element of the ring at `0`. With `new` I would have an empty ring with some capacity

Reply:
Make sense